### DISCUSSION TITLE
AWS Actions disappeared from github.com/actions/aws

### DISCUSSION THREAD

**Discussion by f-barth** [community/community#25066] [Actions]
<p>Hi all, We have been ( :confused: ) running some Actions that relied on <a href="https://github.com/actions/aws" rel="nofollow noopener">https://github.com/actions/aws</a> . This repository did disappear from the “actions” organisation an leads to a lot of broken builds and unhappy DevOps rangers over here :slight_smile: Do you guys have some insights on the why? “on purpose” vs “by accident” If “on purpose”: Can you recommend any alternatives? Is there any communication channel to be informed of the removal of repositories? Thank you, Cheers, Florian</p>

**Comment by chrispat** [comment, marked as answer]
<p>As was announced in this change log <a href="https://github.blog/changelog/2019-12-10-github-actions-aws-and-gcloud-actions-deprecated/" rel="nofollow noopener">https://github.blog/changelog/2019-12-10-github-actions-aws-and-gcloud-actions-deprecated/</a> as well as in a message in the log and in the read me for the action. Both the actions/aws and actions/gcloud actions are deprecated in favor of the official actions from Amazon and Google respectively.</p>

**Comment by f-barth** [comment]
<p>Thank you, <a class="mention" href="/chrispat">@chrispat</a></p>

**Comment by f-barth** [comment]
<p>Do you know if the archived content of actions/aws is still available? <a class="mention" href="/chrispat">@chrispat</a></p>

**Comment by chrispat** [comment]
<p>Sure the content is still around in a private repo but the action did not provide any additional value over calling the AWS cli.</p>

**Comment by GregoryWiltshire** [comment]
<p>For anyone else who finds themselves with this breaking their actions, I have a team member who is maintaining an action which includes awscli and a few other useful utilities.</p>
<p><a href="https://github.com/turnerlabs/fargate-cicd-action" rel="nofollow noopener">https://github.com/turnerlabs/fargate-cicd-action</a></p>
<p>Atlassian also has an image which would be useful for CICD:<br>
<a href="https://hub.docker.com/r/atlassian/pipelines-awscli" rel="nofollow noopener">https://hub.docker.com/r/atlassian/pipelines-awscli</a></p>
